package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "script-loader",
-  "version": "0.7.2",
+  "version": "0.7.3",
```

Comment:
Please revert this as it is the maintainers responsibility to do this

Reply:
sorry, my bad, will do